flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/table/AggregateITCase.scala

```diff
@@ -307,11 +308,16 @@ class AggregateITCase extends StreamingWithStateTestBase {
     data.+=((4, 3L, "C"))
     data.+=((5, 3L, "C"))
 
+    tEnv.registerTableSink(
+      "testSink",
+      new TestUpsertSink(Array("c"), false).configure(
+        Array[String]("c", "bMax"), Array[TypeInformation[_]](Types.STRING, Types.LONG)))
+
     val t = env.fromCollection(data).toTable(tEnv, 'a, 'b, 'c)
             .groupBy('c)
-            .select('c, 'b.max)
+            .select('c, 'b.max as 'bMax)
```

Comment:
Do we have to add alias ?

Reply:
No, you are right. I assumed that field names need to match.